partitionmanager/cli.py

```diff
@@ -273,6 +273,56 @@ def migrate_cmd(args):
 )
 MIGRATE_PARSER.set_defaults(func=migrate_cmd)
 
+def _partition_table(conf, log, table, metrics):
+    table_problems = pm_tap.get_table_compatibility_problems(conf.dbcmd, table)
+    if table_problems:
```

Comment:
```suggestion
    if table_problems := pm_tap.get_table_compatibility_problems(conf.dbcmd, table):
```
https://pypi.org/project/auto-walrus

Reply:
WALRUS!